packages/http-client-csharp/generator/Microsoft.Generator.CSharp/test/CSharpGenTests.cs

```diff
@@ -62,18 +62,18 @@ public void VisitorsAreVisited()
             mockVisitor.Verify(m => m.Visit(mockPlugin.Object.OutputLibrary), Times.Once);
         }
 
-        private void TestOutputPathAppended(string outputPath, string expectedPath)
-        {
-            var srcPath = "src";
+        //private void TestOutputPathAppended(string outputPath, string expectedPath)
```

Comment:
intentional comment?

Reply:
Yeah these test cases are moved to a new place